### PR TITLE
Add Response default property value checker

### PR DESCRIPTION
**Pull request by blva**
- Add test for response property format updates
- Add checker and tests for response default property and media type default value updates

### CODE REVIEW

**Review by ciprian-tibulca — APPROVED** [review]

File: checker/localizations_src/en/messages.yaml [hunk 1]
```diff
@@ -160,3 +160,6 @@ response-optional-property-became-write-only: the response optional property %s
 response-optional-property-became-read-only: the response optional property %s became read-only for the status %s
 response-optional-property-became-not-write-only: the response optional property %s became not write-only for the status %s
 response-optional-property-became-not-read-only: the response optional property %s became not read-only for the status %s
+response-property-default-value-changed: the response property %s default value changed from %s to %s for the status %s
+response-media-default-value-changed: the response property %s default value changed from %s to %s for the status %s
```
Comment (reuvenharrison) [inline checker/localizations_src/en/messages.yaml]: unused

File: checker/check-response-property-default-value-changed_test.go [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+		Comment:     "",
+		Level:       checker.INFO,
+		Operation:   "POST",
+		Path:        "/api/v1.0/groups",
+		Source:      "../data/checker/response_property_default_value_changed_revision.yaml",
+		OperationId: "createOneGroup",
+	}, errs[0])
+}
+
+// CL: Changing response media type default value
```
Comment (reuvenharrison) [inline checker/check-response-property-default-value-changed_test.go]: maybe: "Changing response body default value"?

File: checker/check-response-property-default-value-changed.go [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+				}
+
+				modifiedMediaTypes := responseDiff.ContentDiff.MediaTypeModified
+				for mediaType, mediaTypeDiff := range modifiedMediaTypes {
+					if mediaTypeDiff.SchemaDiff != nil && mediaTypeDiff.SchemaDiff.DefaultDiff != nil {
+						defaultValueDiff := mediaTypeDiff.SchemaDiff.DefaultDiff
+						result = append(result, ApiChange{
+							Id:          "response-body-default-value-changed",
+							Level:       INFO,
+							Text:        fmt.Sprintf(config.i18n("response-body-default-value-changed"), ColorizedValue(mediaType), empty2none(defaultValueDiff.To), empty2none(defaultValueDiff.To), ColorizedValue(responseStatus)),
```
Comment (reuvenharrison) [inline checker/check-response-property-default-value-changed.go]: first empty2none(defaultValueDiff.To) should be empty2none(defaultValueDiff.From)

Comment (blva) [reply checker/check-response-property-default-value-changed.go]: nice catch thanks, updating

**Review by reuvenharrison — CHANGES_REQUESTED** [review]
I see you didn't add the new error messages in Russian.
While this still works, by falling back to English, I recommenf either of the following alternatives:
1. Add the Russian messages
2. Document the missing Russian messages as a TODO task
3. Automatically find missing entries in any of the languages and break the build

**Review by reuvenharrison — APPROVED** [review]

**Pull request merged**